pom.xml

```diff
@@ -61,16 +61,25 @@ THE SOFTWARE.
 		<scmTag>HEAD</scmTag>
 		<gitHubRepo>jenkinsci/horreum-plugin</gitHubRepo>
 
-		<jenkins.version>2.332.3</jenkins.version>
+		<!-- 2.361 -->
+		<jenkins.version>2.440.3</jenkins.version>
+		<hpi-plugin.version>3.37</hpi-plugin.version>
 		<forkCount>1C</forkCount>
-		<jackson.version>2.14.1</jackson.version>
+		<jackson.version>2.17.2</jackson.version>
 		<access-modifier.version>1.25</access-modifier.version>
-		<testcontainers.version>1.17.6</testcontainers.version>
 		<junit.jupiter.version>5.9.1</junit.jupiter.version>
-		<horreum.version>0.7</horreum.version>
+		<horreum.version>0.15.2</horreum.version>
+		<keycloak.version>25.0.2</keycloak.version>
+		<maven.compiler.target>17</maven.compiler.target>
+		<maven.compiler.source>17</maven.compiler.source>
```

Comment:
Forcing Java 17 means that users running Java 11 on Jenkins 2.440.3, 2.452.x, and 2.462.x will fail to run the plugin.

If you must force Java 17, then you should probably set the base Jenkins version to Jenkins 2.477 weekly or newer.  Jenkins LTS core will require Java 17 or newer with the 30 Oct 2024 release of Jenkins 2.477.1.
```suggestion
```

Reply:
@MarkEWaite Thanks for spotting this!

@whitingjr the plugin should target Java 11

Reply:
@johnaohara it now targets 11 and uses 0.15.3 release of Horreum artifacts.